huaweicloud/services/oms/resource_huaweicloud_oms_migration_task_group.go

```diff
@@ -0,0 +1,835 @@
+							ForceNew:  true,
+							RequiredWith: []string{
+								"source_object.0.access_key",
+							},
+						},
+						"object": {
+							Type:     schema.TypeList,
+							Optional: true,
+							ForceNew: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+							ExactlyOneOf: []string{
+								"source_object.0.list_file_bucket",
+							},
+						},
+						"data_source": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Default:  "Aliyun",
+						},
+						"app_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"list_file_bucket": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							RequiredWith: []string{
+								"source_object.0.list_file_key",
+							},
+						},
+						"list_file_key": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							RequiredWith: []string{
+								"source_object.0.list_file_bucket",
+							},
+						},
+					},
+				},
+			},
+			"destination_object": {
+				Type:     schema.TypeList,
+				Required: true,
+				ForceNew: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"region": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"bucket": {
+							Type:     schema.TypeString,
+							Required: true,
+							ForceNew: true,
+						},
+						"access_key": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+						"secret_key": {
+							Type:      schema.TypeString,
+							Sensitive: true,
+							Optional:  true,
+							ForceNew:  true,
+						},
+						"data_source": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+							Default:  "HEC",
+						},
+						"save_prefix": {
+							Type:     schema.TypeString,
+							Optional: true,
+							ForceNew: true,
+						},
+					},
+				},
+			},
+			"type": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"enable_kms": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"migrate_since": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"object_overwrite_mode": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"consistency_check": {
+				Type:     schema.TypeString,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"enable_requester_pays": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				ForceNew: true,
+				Computed: true,
+			},
+			"action": {
+				Type:     schema.TypeString,
+				Optional: true,
```

Comment:
I think we can add a validation for the value because we also check the value in handleMigrationTaskGroupAction